content/docs/04-clusters/02-data-center/01-maas/15-create-manage-maas-clusters.md

```diff
@@ -118,18 +122,27 @@ The cluster status is updated to **Deleting** while cluster resources are being
 
 <br />
 
-## Force Delete a Cluster
 
-If a cluster is stuck in the Deletion state for a minimum of 15 minutes it becomes eligible for force deletion. You can force delete a cluster from the tenant and project admin scopes. To force delete a cluster, follow the same steps outlined in [Delete a MAAS Cluster](/clusters/data-center/maas/create-manage-maas-clusters#deleteamaascluster). After 15 minutes, a **Force Delete Cluster** option is available in the **Settings drop-down menu**. The drop-down menu will provide you with an estimated remaining duration before the force deletion becomes available.
+# Upgrading a MAAS Cluster
 
-<br />
+Upgrading a cluster involves careful planning and executing the upgrade while minimizing disruptions to your applications and services. Spectro Cloud provides an intuitive and streamlined upgrade process through the MAAS, simplifying the entire process.
+
+To learn more about managing a cluster, refer to [Manage Clusters](https://docs.spectrocloud.com/clusters/cluster-management/cluster-updates).
 
 <WarningBox>
 
-A force delete can result in Palette-provisioned resources being missed during the removal process. Verify that any MAAS machines associated with the cluster have been released. Failure to remove provisioned resources can result in unexpected costs.
+When you use a new pack for a specific OS version with MAAS, the absence of the corresponding base image for that OS version in your MAAS configuration can cause errors in Palette.
+
+Ensure you have obtained and imported the correct base image for the required OS version in the MAAS management interface to fix this problem. By doing this, you can eliminate issues and successfully use the Palette pack for the chosen OS version.
+
+Store the corresponding base image in MAAS to enable seamless integration with the desired OS version and prevent compatibility issues.
```

Comment:
Use simpler language, and shorten this. We don't want long paragraphs in a box element. Also, link to the download page docs from MaaS.

Reply:
Hi Karl, I have shortened it. Could you let me know once if it's okay? I have added the required information the user needs to avoid generating the error.